scripts/build_package.py

```diff
@@ -709,7 +713,14 @@ def build_package(repository_root, build_package_config, env=None):
     LOG.debug('Copy web client files')
     source = os.path.join(repository_root, 'www')
     target = os.path.join(package_root, package_layout['www'])
-    copy_tree(source, target)
+    userguide_images = os.path.join(source, 'userguide', 'images')
+    copy_tree(source, target, [userguide_images])
+    copy_tree(userguide_images, os.path.join(target, 'images'))
+
+    # Run doxygen.
```

Comment:
A separate make file target would be better, or please extend the current `docs` target to generate html from the markdown.

Reply:
I agree here. Even if it requires a separate Doxyfile to exist. Also, the `build_script.py` does not run in parallel, while `make` can make the build distributed if multiple threads are allowed. (Not that documentation generation should be paralellised so much, I know.)